libvast/vast/system/import_command.hpp

```diff
@@ -65,11 +64,14 @@ caf::expected<expression> parse_expression(command::argument_iterator begin,
 
 } // namespace
 
-template <class Reader, class Defaults>
-caf::message import_command(const invocation& inv, caf::actor_system& sys) {
-  VAST_TRACE(inv.full_name, VAST_ARG("options", inv.options), VAST_ARG(sys));
+template <class Reader, class Defaults,
+          caf::spawn_options SpawnOptions = caf::spawn_options::no_flags,
+          class Actor>
+caf::expected<caf::actor>
+configure_source(const Actor& self, caf::actor_system& sys,
+                 const invocation& inv, accountant_type accountant,
+                 type_registry_type type_registry, caf::actor importer) {
```

Comment:
Can you add a comment describing what this function does? I also think that the name is not indicative of the effect. Maybe just `spawn_source_impl`?

Reply:
What about `make_source`?

Reply:
I went with `make_source` for now and moved it into its own file.

Reply:
I'm OK with that.